### PR TITLE
detect-pcre/v1: remove unittests

### PR DESCRIPTION
**Pull request by TheKharleeci**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

Describe changes:
- remove unittests that have been converted to Suricata-Verify

suricata-verify-pr: 717

### CODE REVIEW

File: src/detect-pcre.c [hunk 1]
```diff
@@ -1654,1078 +1654,56 @@ static int DetectPcreParseTest28(void)
...
-                    "lalala lalala\\ lala\n";
+    uint8_t *buf = (uint8_t *)"lalala lalala\\ lala\n";
     uint16_t buflen = strlen((char *)buf);
-    Packet *p = UTHBuildPacket( buf, buflen, IPPROTO_TCP);
+    Packet *p = UTHBuildPacket(buf, buflen, IPPROTO_TCP);
     int result = 0;
 
-    char sig[] = "alert tcp any any -> any any (msg:\"pcre with an ending slash\"; pcre:\"/ lalala\\\\/\"; sid:1;)";
+    char sig[] = "alert tcp any any -> any any (msg:\"pcre with an ending slash\"; pcre:\"/ "
+                 "lalala\\\\/\"; sid:1;)";
```
Comment (inashivb) [inline src/detect-pcre.c:1665]: Was this clang-format?

**Review by inashivb — COMMENTED** [review]
LGTM on the first sight. But, the diff here is not making things easy and given that this is a big PR, I'll request an extra pair of eyes
^^ @jufajardini 

Also, @TheKharleeci we should probably also mention in this commit that the tests were converted to s-v tests just like all other commits you've made ;)

File: src/detect-pcre.c [hunk 2]
```diff
@@ -3576,36 +2236,11 @@ static void DetectPcreRegisterTests(void)
...
-
-    UtRegisterTest("DetectPcreModifPTest04 -- Modifier P",
-                   DetectPcreModifPTest04);
-    UtRegisterTest("DetectPcreModifPTest05 -- Modifier P fragmented",
-                   DetectPcreModifPTest05);
     UtRegisterTest("DetectPcreTestSig06", DetectPcreTestSig06);
     UtRegisterTest("DetectPcreTestSig07 -- anchored pcre",
                    DetectPcreTestSig07);
     UtRegisterTest("DetectPcreTestSig08 -- anchored pcre",
                    DetectPcreTestSig08);
```
Comment (jufajardini) [inline src/detect-pcre.c:2243]: Can these be renamed to reflect the removal of the previous related tests?

**Review by jufajardini — CHANGES_REQUESTED** [review]
So many lines removed :D 🎉 

One nit comment (about renaming tests, see inline)

And two considerations that relate to how some of the tests are being covered (or not) by the corresponding S-V tests proposed, so we make sure to check that, before we remove them.

**Comment by TheKharleeci** [comment]
Followed By: #7109

**Pull request closed without merging**